build.gradle

```diff
@@ -101,15 +102,15 @@ jacocoTestCoverageVerification {
             limit {
                 counter = 'METHOD'
                 value = 'COVEREDRATIO'
-                minimum = 0.800
+                minimum = 0.87
```

Comment:
For my sake, how are these ratios decided on and why?

Reply:
In this case, it is just me upping the ratios as they improve — ideally 1.000, but as is, the code is only at 87%